js/apps/admin-ui/CONTRIBUTING.md

```diff
@@ -14,6 +14,8 @@ You can find out which version of Node.js you are using by running the following
 node --version
 ```
 
+For the `pnpm` command to be available, run the command `corepack enable` once. 
```

Comment:
Perhaps we can simply copy this documentation that I wrote for one of my repos: https://github.com/jonkoops/kjs-playground?tab=readme-ov-file#setup

Reply:
Hm, its beyond my skills to identify what to copy.